### PR TITLE
Create style.css

### PR DESCRIPTION
**Pull request by RhyenSftwareEngr**
Pls put codes here

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (AVILLANOZA) [inline style.css:1]: code reviewww

**Comment by eurecho** [comment]
oks

**Review by grgn2002 — COMMENTED** [review]
Code = good

File: style.css (hunk 1 above)
Comment (eurecho) [reply style.css:1]: oks

**Review by eurecho — COMMENTED** [review]
gg

**Pull request merged**